openupgrade_scripts/scripts/project/15.0.1.2/pre-migration.py

```diff
@@ -0,0 +1,14 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+            ALTER TABLE project_project
+            ADD COLUMN IF NOT EXISTS last_update_status CHARACTER VARYING;
+            UPDATE project_project
+            SET last_update_status = 'last_update_status';
```

Comment:
@hoangtiendung070797 lỗi chính tả à?
```suggestion
            SET last_update_status = 'on_track';
```

Reply:
@royleviindoo dạ vâng anh, em k để ý đoạn này